src/components/Table/TableHead/_table-head.scss

```diff
@@ -0,0 +1,21 @@
+@import '~carbon-components/scss/globals/scss/vars';
+@import '~carbon-components/scss/globals/scss/layout';
+
+.#{$prefix}--data-table {
+  .#{$prefix}--table-header-label {
+    display: flex;
+  }
+
+  .column-header__btn {
+    margin: 0 $spacing-05 $spacing-05 0;
+    max-height: $spacing-07;
```

Comment:
I am still not a fan of using unrelated tokens in place of values. If this spacing token gets overwritten either by Carbon or consuming teams this max-height will change.

Reply:
Yeah, I agree. In this case, this same token is used for the `min-height` in the Carbon styles. For whatever reason, there isn't a `(max-)height` set, only a `min-height`. If this token changes in the future, we'll have to touch these styles regardless.